source/DSCResources/MSFT_ADFineGrainedPasswordPolicy/en-US/about_ADFineGrainedPasswordPolicy.help.txt

```diff
@@ -0,0 +1,146 @@
+.NAME
+    ADFineGrainedPasswordPolicy
+
+.DESCRIPTION
+    The ADFineGrainedPasswordPolicy DSC resource will manage an Active Directory domain fine grained password policy.
+
+    ## Requirements
+
+    * Target machine must be running Windows Server 2012 or later.
+
+.PARAMETER Name
+    Key - String
+    Specifies an Active Directory fine-grained password policy object name.
+
+.PARAMETER Precedence
+    Required - UInt32
+    Specifies a value that defines the precedence of a fine-grained password policy among all fine-grained password policies.
+
+.PARAMETER DisplayName
+    Write - String
+    Specifies the display name of the object.
+
+.PARAMETER Subjects
+    Write - String[]
+    Specifies the ADPrincipal names the policy is to be applied to, overwrites all existing.
+
+.PARAMETER Ensure
+    Write - String
+    Allowed values: Present, Absent
+    Specifies whether the fine-grained password policy should be present or absent. Default value is 'Present'.
+
+.PARAMETER ComplexityEnabled
+    Write - Boolean
+    Specifies whether password complexity is enabled for the password policy.
+
+.PARAMETER LockoutDuration
```

Comment:
Please adjust the `LockoutDuration`, `LockOutObservationWindow`, `MinPasswordAge` and `MaxPasswordAge` descriptions to the new ones specified in the schema.